detection-rules/link_new_domain_in_link_first_time_sender.yml

```
@@ -0,0 +1,22 @@
+name: "Link: New domain (<7d) in link from first-time sender"
+description: |
+  Detects links in the body of an email where the linked domain is less than 7 days old.
+type: "rule"
+severity: "high"
```

```suggestion
severity: "medium"
```

thoughts on medium? i think if we combined this with other clearly bad stuff it could definitely be high